examples/plot_goes_hek_m25.py

```diff
@@ -0,0 +1,32 @@
+"""
+===============================
+GOES Flare and HEK Plot Example
+===============================
+
+Here is an example showing how to combine GOES and HEK data
+"""
+
+import matplotlib.pyplot as plt
+from sunpy.lightcurve import GOESLightCurve
+from sunpy.time import TimeRange, parse_time
+from sunpy.net import hek
+
+###############################################################################
+# Let's first grab GOES XRS data for a particular time of interest
+
+tr = TimeRange(['2011/06/07 04:00', '2011/06/07 12:00'])
+goes = GOESLightCurve.create(tr)
+
+###############################################################################
+# Next lets grab the HEK data for this time from the NOAA Space Weather Prediction Center (SWPC)
+
+client = hek.HEKClient()
+result = client.query(hek.attrs.Time(tr.start,tr.end), hek.attrs.FL, hek.attrs.FRM.Name == 'SWPC')
```

Comment:
wouldn't be a variable name like `flares_hek` or similar more appropriate than `result`?